app/main.py

```diff
@@ -1,3 +1,13 @@
 def cache(func):
-    # Write your code here
-    pass
+    cache_list = []
+
+    def inner(*args, **kwargs):
+        if args in cache_list:
+            print("Getting from cache")
+            return func(*args, **kwargs)
```

Comment:
The function executes func for each call. The main idea is to somehow store the results for previous arguments and return them for calls with the same arguments without executing func.